autowarefoundation/autoware.universe
# fix(scan_ground_filter): fix segmentation fault

## Description


Fix segmentation fault due to access violation in out_radial_ordered_points[radial_div]

### Test procedure
Run rosbag replay simulation with sample-bag and other rosbags to verify that the scan ground filter is not terminated.
Tested with rosbag, which was causing the segmentation fault, and confirmed that there was no problem.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.


## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.


## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.


After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/
